shared/team-building/phone-search.tsx

```diff
@@ -54,7 +54,7 @@ const PhoneSearch = (props: PhoneSearchProps) => {
     setPhoneNumber('')
     setPhoneInputKey(old => old + 1)
     setPhoneValidity(false)
-  }, [dispatch, namespace, user, phoneNumber, setPhoneNumber, canSubmit, setPhoneInputKey])
+  }, [dispatch, namespace, user, setPhoneNumber, canSubmit, setPhoneInputKey])
```

Comment:
@zapu please validate

Reply:
doesn't have to dep on `setPhoneValidity`?

Reply:
yeah looks like `setPhoneValidity` needs to be there too